### PR TITLE
Added email and message inputs to footer

### PR DESCRIPTION
**Pull request by Cameron-Porter**
## Description
Took the contact form and edited a copy in order to match the screenshot given as a guide for the form. Then ensured it was imported in the footer, extended the length of the name input, and added ContactFormFooter to all appropriate index.js files. Also changed up the label working to match the photo in the issue.

Still needs to have the button centered and the 15px padding-left removed. Also needs a live word count 0/300 below the texture.

## Related Issue
#259 

## Motivation and Context
The Spring 2021 cohort implemented this feature and it was desired to have it brought onto the app.

## How Has This Been Tested?
The functionality still needs confirmation of message being sent.
nom run test is good to go.

## Screenshots (if appropriate):
Screen Shot 2022-04-07 at 8 47 30 PM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/components/Forms/ContactFormFooter/ContactFormFooter.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      const contactApiEndpoint = '/api/contact'
+      const options = {
+        method: 'POST',
+        body: JSON.stringify(formData),
+      }
+      const response = await fetch(contactApiEndpoint, options)
+      if (response.ok) {
+        onSubmitSuccess('Your form was successfully submitted.')
+        setLoading(false)
+        setPhone('')
```
Comment (andrew-t-james) [inline src/components/Forms/ContactFormFooter/ContactFormFooter.js:24]: is `setPhone` Needed?

**Review by andrew-t-james — CHANGES_REQUESTED** [review]
Thanks for PR. Overall looks good. Please add unit test coverage for this feature.

File: src/components/Forms/ContactFormFooter/ContactFormFooter.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+          <label htmlFor="InputEmail" className="light-text">
+            Email
+            <sup>*</sup>
+          </label>
+          <input
+            type="email"
+            className="form-control"
+            name="email"
+            id="InputEmail"
+            placeholder="cam@example.com"
```
Comment (andrew-t-james) [inline src/components/Forms/ContactFormFooter/ContactFormFooter.js:46]: Please use a similar/same placeholder that the other email inputs use. See example [here](https://github.com/Vets-Who-Code/vets-who-code-app/blob/ad219f60247ba5ae795d16c4e4847a48b23ee1af/src/components/Forms/ContactForm/ContactForm.js#L82)

**Review by andrew-t-james — CHANGES_REQUESTED** [review]

File: src/components/Forms/ContactFormFooter/ContactFormFooter.js (hunk 2 above)
Comment (Cameron-Porter) [reply src/components/Forms/ContactFormFooter/ContactFormFooter.js:46]: Was having issues for some reason with pushing changes from this branch back after updating. Created a new branch with changes and set the branch to merge into this one. Added @andrew-t-james as a reviewer.

**Pull request closed without merging**